cms/static/cms/js/modules/cms.plugins.js

```diff
@@ -183,6 +186,26 @@ var CMS = window.CMS || {};
                     );
                 });
 
+                // prevents single click from messing up the edit call
+                this.ui.container.find('a').on('click', function (e) {
+                    if (e.shiftKey || e.ctrlKey || e.metaKey) {
+                        return;
+                    }
+
+                    e.preventDefault();
```

Comment:
we can also check if the `e.isDefaultPrevented()` and immediately bail out right here by returning
this way if the user has a custom js on a link with the url and he does the preventing already - we don't go to the link